content/hi/docs/tasks/tools/included/kubectl-convert-overview.md

```diff
@@ -0,0 +1,11 @@
+---
+title: "कुबेक्टल-कन्वर्ट ओवरव्यू"
+description: >-
+  एक Kubectl प्लगइन जो आपको मैनिफेस्ट को एक संस्करण से बदलने की अनुमति देता है
+  एक कुबेरनेट्स एपीआई के एक अलग संस्करण के लिए।
+headless: true
+---
+
+कुबेरनेट्स कमांड-लाइन टूल `kubectl` के लिए एक प्लगइन, जो आपको विभिन्न API संस्करण के बीच मैनिफ़ेस्ट को कनवर्ट करने की अनुमति देता है।
+यह विशेष रूप से नए कुबेरनेट्स रिलीज के साथ एक गैर-बहिष्कृत एपीआई संस्करण में मैनिफेस्ट को माइग्रेट करने में सहायक हो सकता है।
+अधिक जानकारी के लिए, [migrate to non deprecated apis](/docs/reference/using-api/deprecation-guide/#migrate-to-non-deprecated-apis) पर जाएं।
```

Comment:
Thanks, @ShivamTyagi12345, try to find better Hindi localization for this too
```suggestion
अधिक जानकारी के लिए, [गैर पदावनत एपिस में विस्थापित करें](/docs/reference/using-api/deprecation-guide/#migrate-to-non-deprecated-apis) पर जाएं।
```

Reply:
Yeah would go ahead and open a slack thread